src/client/components/Pagination/Pagination.stories.js

```diff
@@ -29,16 +30,13 @@ const template = (args) => {
         </li>
       ))}
       {/* eslint-disable */}
-      <PaginationComponent
-        onPageChange={(range) => setCurrentRange(range)}
-        {...args}
-      />
+      <Pagination onPageChange={(range) => setCurrentRange(range)} {...args} />
```

Comment:
We need to give pagination the props the it require like this: 
`<Pagination
        currentPageIndex={currentPageIndex}
        setCurrentPageIndex={setCurrentPageIndex}
        pageCount={args.products.length / args.productsPerPage}
        {...args}
      />`

Reply:
Changed like you mentioned...